libcloud/test/compute/test_ecs.py

```diff
@@ -539,13 +546,36 @@ def test_ex_leave_security_group(self):
                                                      group_id=ex_security_group_id_value)
         self.assertTrue(result)
 
+    def test_ex_delete_security_group_by_id(self):
+        ex_security_group_id_value = "sg-28ou0f3xa"
+        result = self.driver.ex_delete_security_group_by_id(
+            group_id=ex_security_group_id_value)
+        self.assertTrue(result)
+
+    def test_ex_modify_security_group_by_id(self):
+        ex_security_group_id_value = "sg-28ou0f3xa"
+        result = self.driver.ex_modify_security_group_by_id(
+            group_id=ex_security_group_id_value)
+        self.assertTrue(result)
+
```

Comment:
It would be awesome if you could specify a name and description in those tests to improve coverage!